tests/sparseml/onnx/optim/test_analyzer_model.py

```diff
@@ -137,38 +140,43 @@ def test_mode_analyzer_json():
     )
 
 
-def test_model_analyzer(analyzer_models):  # noqa: F811
-    model_path, *expected_outputs = analyzer_models
+@pytest.mark.parametrize(
+    "cls,fname",
+    [
+        (LinearNet, "LinearNet-single.json"),
+        (LinearNet, "LinearNet-batched.json"),
+        (MLPNet, "MLPNet-single.json"),
+        (ConvNet, "ConvNet-batched.json"),
+    ],
+)
+def test_model_analyzer(tmp_path, cls, fname):  # noqa: F811
```

Comment:
@bfineran @rahul-tuli thoughts on moving these tests to having a fixed onnx graph that it analyzes? Currently these test are testing two things:
1. ModuleExporter (which is why we need different versions of the files)
2. ModuleAnalyzer

Removing the ModuleExporter part and moving to analyzing a fixed onnx graph will
1. Make this much simpler (we can remove the need for maintaining different versions)
2. make the unit test much more focused

I guess this hinges on having other tests covering ModuleExporter. Do you think we have that?